src/skins/blau.tsx

```diff
@@ -171,6 +171,10 @@ export const getBlauSkin: GetKnownSkin = () => {
             tagBackgroundSuccess: palette.blauGreen10,
             tagBackgroundWarning: palette.blauYellow10,
             tagBackgroundError: palette.blauRed10,
+            cardContentOverlay: `linear-gradient(180deg, ${applyAlpha(palette.grey6, 0)} 0%, ${applyAlpha(
```

Comment:
Do we have a ticket to apply this new token @yceballost ?

Reply:
yes! https://jira.tid.es/browse/WEB-1799